apps/testing-javascript/README.md

```diff
@@ -336,7 +336,42 @@ Start prisma studio:
 pnpm db:studio
 ```
 
-### Seed Data
+### Seed Data the First Time
```

Comment:
Add details to the README about how to apply the `.sql` seed files in `seed_data/`